compiler/resolution/generics.cpp

```diff
@@ -258,7 +258,13 @@ instantiate_tuple_autoCopy(FnSymbol* fn) {
     Symbol* tmp = newTemp();
     block->insertAtTail(new DefExpr(tmp));
     block->insertAtTail(new CallExpr(PRIM_MOVE, tmp, new CallExpr(PRIM_GET_MEMBER_VALUE, arg, new_StringSymbol(astr("x", istr(i))))));
-    call->insertAtTail(new CallExpr("chpl__autoCopy", tmp));
+    // If it is a reference, pass it through.
```

Comment:
It is interesting to note that I had already effectively made this change on the string-as-rec branch.  AMM later decides whether an autoCopy is needed.  In the case of a ref, it decides "no".  So in fact, after AMM is merged, the conditional can be removed and replaced by just the "true" branch.
Not to worry: this choice will be obvious at the time the merge is performed.  (I have put the new code in a HILDE_MM conditional compilation clause.)